### PR TITLE
Commerce-10367 Integration test for Account Defaults API

### PR DESCRIPTION
**Pull request by smottal**
https://issues.liferay.com/browse/COMMERCE-10367

Resent from https://github.com/liferay-commerce/liferay-portal/pull/3241

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-test/src/testIntegration/java/com/liferay/headless/commerce/admin/account/resource/v1_0/test/AccountChannelEntryResourceTest.java [hunk 1]
```diff
@@ -15,15 +15,1142 @@
...
+
+		return _postAccountChannelEntryDiscount();
+	}
+
+	@Override
+	protected AccountChannelEntry
+			testPatchAccountChannelPaymentTermId_addAccountChannelEntry()
+		throws Exception {
+
+		return _postAccountChannelEntryPaymentTerm();
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-test/src/testIntegration/java/com/liferay/headless/commerce/admin/account/resource/v1_0/test/AccountChannelEntryResourceTest.java:782]: My opinion, following Brian review is that you don't need these methods too.

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
ehi @smottal please change also these methods.
Thanks!

**Review by andrea-ale-sbarra — APPROVED** [review]
on a second glance I think these methods are good enough!

**Comment by smottal** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**